ui/app/src/modules/Content/Publish/PublishDialog.tsx

```diff
@@ -385,11 +386,13 @@ function PublishDialogWrapper(props: PublishDialogProps) {
   const siteId = useActiveSiteId();
   const permissionsBySite = usePermissionsBySite();
   const myPermissions = permissionsBySite[siteId];
+  const dispatch = useDispatch();
 
   useUnmount(props.onClosed);
 
   const user = useSelector<GlobalState, GlobalState['user']>((state) => state.user);
   const submit = myPermissions.includes('publish') ? goLive : submitToGoLive;
+  const propagateAction = myPermissions.includes('publish') ? itemsApproved : itemsScheduled;
```

Comment:
With the publish permission, you may still schedule. Please revisit this logic

Reply:
will check this on other ticket, this behavior is not part of this ticket